ci-operator/step-registry/cucushift/upgrade/disconnected/mirrorimage/cucushift-upgrade-disconnected-mirrorimage-ref.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/upgrade/disconnected/mirrorimage/cucushift-upgrade-disconnected-mirrorimage-ref.yaml",
```

Comment:
suggest rename the path like the following, to keep consistent with fresh install.
```
cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-ref.yaml
```

Reply:
Thanks for the suggestion!